dev/cnf/oss_ibm.maven

```diff
@@ -98,6 +98,11 @@ org.eclipse.persistence:org.eclipse.persistence.core:2.7.7-69f2c2b
 org.eclipse.persistence:org.eclipse.persistence.jpa.jpql:2.7.7-69f2c2b
 org.eclipse.persistence:org.eclipse.persistence.jpa.modelgen:2.7.7-69f2c2b
 org.eclipse.persistence:org.eclipse.persistence.jpa:2.7.7-69f2c2b
+org.eclipse.persistence:org.eclipse.persistence.asm:3.0.0-M1
```

Comment:
Please keep this file in alphabetical order

Reply:
ok